skstore/ts/src/skstore_api.ts

```diff
@@ -235,14 +235,28 @@ export interface NonEmptyIterator<T> {
 
 /**
  * A _Lazy_ Handle on a reactive collection, whose values are computed only when queried
- * using `get`
  */
 export interface LHandle<K extends TJSON, V extends TJSON> {
   /**
-   * Get (and potentially compute) a value of a lazy reactive collection
+   * Get (and potentially compute) all values mapped to by some key of a lazy reactive
+   * collection.
    * @throws {Error} when the key does not exist
    */
-  get(key: K): V;
+  getArray(key: K): V[];
```

Comment:
Maybe I'm confused, but is there actually a distinction between a key "not existing" and mapping to an empty array of values? Looking at the skip code, it seems that there is no throw case for `getArray`, and that the a key "existing" amounts to it mapping to a non-empty array of values.

Reply:
Right, thanks -- copied too much of the docstring there from `get` (which does throw) and should weaken the language for `maybe`/`array` versions.